### PR TITLE
[staging] Correcting variables specifiers in some installation language files

### PR DESCRIPTION
**Pull request by infograf768**
Pull Request for Issue https://github.com/joomla/joomla-cms/issues/30604

### Summary of Changes

As title says.

### Testing Instructions
Can't test as I do not have PHP 8, but there were many errors in these language files which are now corrected.

### CODE REVIEW

File: installation/language/hi-IN/hi-IN.ini [hunk 1]
```diff
@@ -50,14 +50,14 @@ INSTL_DATABASE_USER_LABEL="यूजर का नाम"
...
+INSTL_FTP_DESC="<p>कुछ स्थापना पूरा करने के लिए सर्वर पर एफ़टीपी क्रेडेंशियल्स प्रदान की आवश्यकता हो सकती है. यदि आपको क्रेडेंशियल्स के बिना स्थापना को पूरा करने में कठिनाइयों है तो अपने मेजबान के साथ जाँच करे की क्या यह आवश्यक है.</p> <p>सुरक्षा कारणों से, यह सबसे अच्छा है एक अलग FTP उपयोगकर्ता खाता बना ले Joomla! के स्थापना के साथ हो पूरे वेब सर्वर के लिए नहीं. आपका मेजबान इस के साथ आपकी सहायता कर सकते हैं.</p><strong> टिप्पणी: </strong>यदि आप विंडोज ऑपरेटिंग सिस्टम पर स्थापित कर रहे हैं, एफ़टीपी परत है <strong>नहीं </strong> की आवश्यकता है. "
 INSTL_FTP_ENABLE_LABEL="FTP के परत सक्रिय करें"
 INSTL_FTP_HOST_LABEL="FTP होस्ट "
 INSTL_FTP_PASSWORD_LABEL="FTP पासवर्ड"
 INSTL_FTP_PORT_LABEL="FTP पोर्ट"
 INSTL_FTP_ROOT_LABEL="FTP रूट पथ"
 INSTL_FTP_SAVE_LABEL="FTP पासवर्ड सुरक्षित करना"
-INSTL_FTP_TITLE="FTP के कॉन्फ़िगरेशन (<strong वैकल्पिक class="_QQ_"red"_QQ_"> अधिकांश उपयोगकर्ताओं के इस कदम को छोड़ सकते हैं - अगले प्रेस </ strong> छोड़ें)"
+INSTL_FTP_TITLE="FTP के कॉन्फ़िगरेशन (<strong> वैकल्पिक class="_QQ_"red"_QQ_"> अधिकांश उपयोगकर्ताओं के इस कदम को छोड़ सकते हैं - अगले प्रेस </strong> छोड़ें)"
```
Comment (Quy) [inline installation/language/hi-IN/hi-IN.ini]: Fix `> वैकल्पिक class`

Comment (infograf768) [reply installation/language/hi-IN/hi-IN.ini]: done

**Comment by infograf768** [comment]
@HLeithner 
Should now be OK for PHP 8.

**Comment by HLeithner** [comment]
Thanks

**Pull request merged**